community/containerd/APKBUILD

```diff
@@ -1,23 +1,29 @@
-# Maintainer: Natanael Copa <ncopa@alpinelinux.org>
+# Maintainer: Bernhard J. M. Gruen <bernhard.gruen@googlemail.com>
 pkgname=containerd
-pkgver=1.0.0
-pkgrel=2
+pkgver=1.2.0
+pkgrel=0
+_commit=c4446665cb9c30056f4998ed953e6d4ff22c7c39
 pkgdesc="An open and reliable container runtime"
 url="https://containerd.io/"
 arch="all"
 license="Apache-2.0"
+depends="runc"
 makedepends="go linux-headers btrfs-progs-dev protobuf-dev"
 source="containerd-$pkgver.tar.gz::https://github.com/containerd/containerd/archive/v$pkgver.tar.gz
 	"
 
 builddir="$srcdir"/containerd-$pkgver
 
-build() {
+prepare() {
 	cd "$builddir"
 	mkdir src
 	mv vendor/* src
 	ln -s "$builddir" src/github.com/containerd/containerd
-	GOPATH="$PWD" LDFLAGS="" make VERSION="$pkgver" REVISION="$pkgrel"
+}
+
+build() {
+	cd "$builddir"
+	GOPATH="$PWD" LDFLAGS="" make VERSION="v$pkgver" REVISION="$_commit"
```

Comment:
Why do you use the commit for REVISION? will docker check it?

I am worried that someone who updates this package in future updates `pkgver` but forget to update `_commit`. Package will build and run but will report a lie about the revision.